src/components/controls/controls.tsx

```diff
@@ -16,85 +17,122 @@ import GeoResolution from "./geo-resolution";
+          tooltip={TreeInfo}
+          mobile={mobileDisplay}
+          options={<>
+            <ChooseLayout />
+            <ChooseMetric />
+            <ChooseBranchLabelling />
+            <ChooseTipLabel />
+            <ChooseSecondTree />
+            <ChooseExplodeAttr tooltip={ExplodeTreeInfo} mobile={mobileDisplay} />
+            <ToggleTangle />
+          </>}
+        />
+      }
+
+      {panelsAvailable.includes("measurements") &&
+        <PanelSection
+          panel="measurements"
+          title={t("sidebar:Measurements")}
+          tooltip={MeasurementsInfo}
+          mobile={mobileDisplay}
+          options={<MeasurementsOptions />}
+        />
       }
 
-      {measurementsOn &&
-        <span style={{ marginTop: "10px" }}>
-          <AnnotatedHeader title={t("sidebar:Measurements Options")} tooltip={MeasurementsOptionsInfo} mobile={mobileDisplay}/>
-          <MeasurementsOptions />
-        </span>
+      {/* Prevent the map from being toggled on when a second tree is visible.
+          It is hidden by logic elsewhere.
+      */}
+      {panelsAvailable.includes("map") && !showTreeToo &&
+        <PanelSection
+          panel="map"
+          title={t("sidebar:Map")}
+          tooltip={MapInfo}
+          mobile={mobileDisplay}
+          options={<>
+            <GeoResolution />
+            <TransmissionLines />
+          </>}
+        />
       }
 
-      {mapOn &&
-        <span style={{ marginTop: "10px" }}>
-          <AnnotatedHeader title={t("sidebar:Map Options")} tooltip={MapOptionsInfo} mobile={mobileDisplay}/>
-          <GeoResolution />
-          <TransmissionLines />
-        </span>
+      {panelsAvailable.includes("entropy") &&
+        <PanelSection
+          panel="entropy"
+          title={t("sidebar:Entropy")}
+          tooltip={EntropyInfo}
+          mobile={mobileDisplay}
+        />
       }
 
-      {frequenciesOn &&
-        <span style={{ marginTop: "10px" }}>
-          <AnnotatedHeader title={t("sidebar:Frequency Options")} tooltip={FrequencyInfo} mobile={mobileDisplay}/>
-          <NormalizeFrequencies />
-        </span>
+      {panelsAvailable.includes("frequencies") &&
+        <PanelSection
+          panel="frequencies"
+          title={t("sidebar:Frequency")}
+          tooltip={FrequencyInfo}
+          mobile={mobileDisplay}
+          options={<NormalizeFrequencies />}
+        />
       }
 
       <span style={{ marginTop: "10px" }}>
-        <AnnotatedHeader title={t("sidebar:Animation Options")} tooltip={AnimationOptionsInfo} mobile={mobileDisplay}/>
+        <ControlHeader title={t("sidebar:Animation Options")} tooltip={AnimationOptionsInfo} mobile={mobileDisplay}/>
         <AnimationOptions />
       </span>
 
-      <span style={{ paddingTop: "10px" }} />
-      <AnnotatedHeader title={t("sidebar:Panel Options")} tooltip={PanelOptionsInfo} mobile={mobileDisplay}/>
-      <PanelLayout />
-      <PanelToggles />
       <Language />
```

Comment:
For consistency, we should change the rendering of "Language" to be a `<ControlHeader>`. Minor.

Reply:
Done: 7608d33a3fb278c258ba84c9adf67bf3291fa585